app/components/amount-input-screen/amount-input-screen.stories.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react"
+import { StoryScreen } from "../../../.storybook/views"
+import { ComponentMeta } from "@storybook/react"
+import { MockedProvider } from "@apollo/client/testing"
+import { createCache } from "../../graphql/cache"
+import { AmountInputScreen, AmountInputScreenProps } from "./amount-input-screen"
+import { WalletCurrency } from "../../graphql/generated"
+import mocks from "../../graphql/mocks"
+import {
+  DisplayCurrency,
+  MoneyAmount,
+  WalletOrDisplayCurrency,
+} from "../../types/amounts"
+
+export default {
+  title: "Amount Input Screen",
```

Comment:
why do we need 2 stories book? feels the same between `Amount Input Screen UI` and `Amount Input Screen`

Reply:
They're very similar. I developed the component by first creating the `AmountInputScreenUI`, which had no logic in it, and I was very easily able to test all permutations of it by just passing in different props without having to mock anything. `AmountInputScreen` uses app state, and therefore requires more boiler plate to be able to exercise the same permutations. I suppose `AmountInputScreen` is what we ultimately want to test, so even though there is more boiler plate there it could make sense to remove the `AmountInputScreenUI` stories and build them out in `AmountInputScreen`.